pkg/k8shandler/fluentd.go

```diff
@@ -256,6 +256,11 @@ func newFluentdPodSpec(logging *logging.ClusterLogging, elasticsearchAppName str
 			Operator: v1.TolerationOpExists,
 			Effect:   v1.TaintEffectNoSchedule,
 		},
+		v1.Toleration{
+			Key:			"node.kubernetes.io/disk-pressure",
```

Comment:
formatting
I'm not sure how the toleration works - by default fluentd would not be scheduled to nodes which have disk pressure, and the toleration would allow fluentd to be scheduled to those nodes?  That is, what does a user do to get fluentd scheduled to nodes where it needs to clear the buffer backlog?
We also need this for rsyslog

Reply:
Yes, this allows it to tolerate that taint so that it can be scheduled to those nodes too.

Reply:
Added rsyslog as well